### ISSUE TITLE
Smart kerning at the end of text

### ISSUE THREAD

**Issue by henryso** [eigilnikolajsen/commit-mono#109]
I've noticed that smart kerning does not occur at the end of the text. For example, `MM` has less space between the M's than `MM ` (that is MM followed by a blank). This is surely an edge case, probably minor, and may be technically infeasable/undesirable.

**Comment by eigilnikolajsen** [comment]
Good call! This is something that I've looked into before, but it's not technically feasible due to the limitations of detecting end of line when using OpenType Features - at least to my knowledge. I will try to improve with the next version if possible:)

**Comment by AmeroHan** [comment]
Sorry for the lack of technical knowledge, but I was wondering if it is difficult to detect end-of-line, is it easier to detect non-end-of-line? Would considering “this letter is at the end of the line” as the default when laying out a letter be able to solve the problem?
